appeals/web/src/server/lib/mappers/appeal/submappers/agent.mapper.js

```diff
@@ -0,0 +1,37 @@
+import { permissionNames } from '#environment/permissions.js';
+import { formatServiceUserAsHtmlList } from '#lib/service-user-formatter.js';
+import { mapActionComponent } from '../../component-permissions.mapper.js';
+
+/**
+ * @typedef {import('#appeals/appeal-details/appeal-details.types.js').WebAppeal} WebAppeal
+ * @typedef {import('../../../../app/auth/auth-session.service.js').SessionWithAuth} SessionWithAuth
+ */
+
+/** @type {import('../appeal.mapper.js').SubMapper} */
+export const mapAgent = ({ appealDetails, currentRoute, session }) => ({
+	id: 'agent',
+	display: {
+		summaryListItem: {
```

Comment:
I suggested to Ben in his related PR that it would be good to add helpers to build these commonly-used GDS component parameters objects (summaryListItem is the most commonly used by far) - just thinking that would reduce the verbosity of many of these individual mappers. I'm not suggesting you do that as part of this PR, it's just a thought.

Reply:
Oh for sure. I didn't read any of the individual mappers truth be told, I'm not sure what most of these do